store/store_download_test.go

```diff
@@ -80,6 +83,21 @@ func (s *storeDownloadSuite) SetUpTest(c *C) {
 			Factor:  1,
 		},
 	)))
+
+	buf, restore := logger.MockLogger()
+	s.AddCleanup(restore)
+	s.logbuf = buf
+
+	os.Setenv("SNAPD_DEBUG", "1")
+	s.AddCleanup(func() { os.Unsetenv("SNAPD_DEBUG")})
+}
+
+func (s *storeDownloadSuite) TearDownTest(c *C) {
```

Comment:
same?

Reply:
logs are actually useful and visible only when run with `-check.vv`

Reply:
fair enough. Also, it's probably not worth changing now but this could also create a logger with the option `LoggerOptions.ForceDebug` instead of setting the env var

Reply:
fair enough. Also, it's probably not worth changing now but this could also create a logger with the option `LoggerOptions.ForceDebug` instead of setting the env var

Reply:
I think it may be useful to have actual helpers to do that, maybe through a new loggertest package?